docs/Prototyping/VCU118.rst

```diff
@@ -58,3 +58,107 @@ The TSI Host Widget is used to interact with the DUT from the prototype over a S
 .. Note:: Remember that since whenever a new test harness is created (or the config changes, or the config packages changes, or...), you need to modify the make invocation.
     For example, ``make SUB_PROJECT=vcu118 CONFIG=MyNewVCU118Config CONFIG_PACKAGE=this.is.my.scala.package bitstream``.
     See :ref:`Prototyping/General:Generating a Bitstream` for information on the various make variables.
+
+Running Linux with Basic and Bringup Platforms
+----------------------------------------------
+
+As mentioned above, the default VCU118 harness is setup with a UART and a SPI SDCard.
+These are utilized to both interact with the DUT (with the UART) and load in Linux (with the SDCard).
+The following steps describe how to build and run buildroot Linux on the prototype platform.
+
+Building Linux with FireMarshal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since the prototype does not have a block device, we build Linux with the rootfs built into the binary (otherwise known as "initramfs" or "nodisk" version of Linux).
```

Comment:
I assume it probably can, but I don't know how to do that (nor do I think it will make much of a difference). Having it as a block device would change things in a couple ways:

1. Have to figure out how to get it to work with Linux / SPI interface. Create new partitions? Have a block device driver? Etc.
2. Could probably minimize the binary size. Thus making loading faster if we change the bootrom.